packages/plugin-vue/src/index.ts

```diff
@@ -201,6 +203,12 @@ export default function vuePlugin(rawOptions: Options = {}): Plugin {
       }
 
       const { filename, query } = parseVueRequest(id)
+
+      // external scoped style
+      if (!query.vue && query.scoped) {
+        return fs.readFileSync(filename, 'utf-8')
+      }
+
       // select corresponding block for sub-part virtual modules
       if (query.vue) {
         if (query.src) {
```

Comment:
BTW, we already supported loading external src import of scoped styles here.
The only problem is that it's coupled with the concept of SFC descriptors during transformation. So I think we only need to fix that part.